manifests/apache.pp

```diff
@@ -53,7 +53,7 @@
   ]
 
   $api_additional_request_headers = $pulpcore::api_client_auth_cn_map.map |String $cn, String $pulp_user| {
-    "set ${remote_user_environ_header} \"${pulp_user}\" \"expr=%{SSL_CLIENT_S_DN_CN} == '${cn}'\""
+    "set ${remote_user_environ_header} \"${pulp_user}\" \"expr=%{tolower:SSL_CLIENT_S_DN_CN} == '${cn.downcase}'\""
```

Comment:
There is another occurence of this in https://github.com/theforeman/puppet-foreman_proxy_content/blob/e24f94194ebd7858f4ce9c34e5a671a30009873c/manifests/container.pp#L26-L27 which I don't understand where it's used from

Reply:
That looks interesting. That code only looks correct for content proxies, not Foreman itself.

Reply:
container gateway is a proxy-only feature :)

Reply:
Oh yes, that isn't `pulpcore::plugin::container`. It's used here: https://github.com/theforeman/puppet-foreman_proxy_content/blob/e24f94194ebd7858f4ce9c34e5a671a30009873c/manifests/init.pp#L254-L256